deploy/03_role.yaml

```diff
@@ -0,0 +1,45 @@
+kind: Role
```

Comment:
I've separated operand and operator roles in different files which may help to track. But fortunately jobset does not require too many permissions.

Reply:
Makes, sense. I will try to improve on that